src/pages/myTodo/tabs/ScrapPage.tsx

```
@@ -1,153 +1,59 @@
         )}
       </div>
-
-      {isModalOpen && <AddressModal onClose={handleRegionSelect} />}
-
-      {selectedRecruitItem && (
-        <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/30"></div>
+      {isModalOpen && (
+        <AddressModal
+          onClose={(region) => {
+            if (region) {
+              activeTab === 'job' ? setJobRegion(region) : setEduRegion(region);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)